Bottom: b175563ffb9eac24f0e251b2d04b0d73f53fe54b
Top:    34c524bbf90c8ef9ccfd7980123b9008a3848a17
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-03-05 15:15:24 +0000

Refresh of remove-left-over-debug

---

diff --git a/src/hid/common/object3d.c b/src/hid/common/object3d.c
index cd1e785..10484d6 100644
--- a/src/hid/common/object3d.c
+++ b/src/hid/common/object3d.c
@@ -1697,8 +1697,6 @@ object3d_from_copper_layers_within_area (POLYAREA *area)
                               copper_appearance,
                               NULL,
                               false)); /* Don't invert */
-
-    //break;
   }
